src/ex5/server/clients/pokemon_client.js

```diff
@@ -1,23 +1,24 @@
-import axios from "axios";
+const axios = require("axios");
 const url = "https://pokeapi.co/api/v2/pokemon";
 let requestsArray = [];
+const TWO_MINS = 2;
```

Comment:
```suggestion
const NUMBER_OF_MINUTES = 2;
```
you want to only change the value if needed without touching the const name
the variable name should be the meaning of the value